### PR TITLE
update documentation to reflect changes in workflow

### PR DESCRIPTION
**Pull request by digorgonzola**
now using GitHub Pages to server static content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 This project is licensed under the terms of the GNU GPLv3 license.
 
 ## Configuration
-This repository contains layer filter configuration for WMS servers defined in the AODN portal (https://github.com/aodn/aodn-portal) as the type 'GeoserverFilterConfig'. The portal expects theses WMS servers to be a standard Geoserver, but will retrieve the filter information from this repo (or S3 repo mirror for prod).
+This repository contains layer filter configuration for WMS servers defined in the AODN portal (https://github.com/aodn/aodn-portal) as the type 'GeoserverFilterConfig'. The portal expects theses WMS servers to be a standard Geoserver, but will retrieve the filter information from this repo from the test branch e.g. `filter_config_url: https://raw.githubusercontent.com/aodn/filter-config/test)`.
```
Comment (nspool) [inline README.md]: Spurious `)` ?

**Review by nspool — CHANGES_REQUESTED** [review]
Spurious `)` ?

File: README.md (hunk 1 above)
Comment (digorgonzola) [reply README.md]: Whoops! Fixed it.

**Review by nspool — APPROVED** [review]
LGTM

**Pull request merged**